netstandard/ref/System.Xml.XPath.cs

```diff
@@ -19,7 +19,6 @@ public partial interface IXPathNavigable
     {
         System.Xml.XPath.XPathNavigator CreateNavigator();
     }
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public static partial class XDocumentExtensions
     {
```

Comment:
Presumably we want to keep hiding this one as the type was a .NET Core 1.x helper that is now replace with the actual instance method on string.

Reply:
This one is actually an additive glue method see http://index/?query=ToXPathNavigable&rightProject=System.Xml.Linq&file=System%5CXml%5CLinq%5CXDocumentExtensions.cs&line=27 so I'd just unhide this in case folks were using it.

Reply:
Cool. I kept them removal.